web/docs/language/features.md

```diff
@@ -443,6 +443,183 @@ import { isPrismaError, prismaErrorToHttpError } from '@wasp/utils.js'
+
+### Recurring jobs
+
+If you have work that needs to be done on some recurring basis, you can add a `schedule` to your job declaration:
+
+```css  {6-9} title="main.wasp"
+job mySpecialJob {
+  executor: PgBoss,
+  perform: {
+    fn: import { foo } from "@ext/jobs/bar.js"
+  },
+  schedule: {
+    cron: "0 * * * *",
+    args: {=json { "job": "args" } json=} // optional
+  }
+}
+```
+
+In this example, you do _not_ need to invoke anything in JavaScript. You can imagine `foo({ "job": "args" })` getting automatically scheduled and invoked for you every hour.
+
+### Fully specified example
+Additionally, both `perform` and `schedule` accept `executorOptions`, which we pass directly to the named job executor when you submit jobs. In this example, the scheduled job will have a `retryLimit` set to 0, as `schedule` overrides any similar property from `perform`.
+
+```css
+job mySpecialJob {
+  executor: PgBoss,
+  perform: {
+    fn: import { foo } from "@ext/jobs/bar.js",
+    executorOptions: {
+      pgBoss: {=json { "retryLimit": 1 } json=}
+    }
+  },
+  schedule: {
+    cron: "*/5 * * * *",
+    args: {=json { "foo": "bar" } json=},
+    executorOptions: {
+      pgBoss: {=json { "retryLimit": 0 } json=}
+    }
+  }
+}
+```
+
+### Fields
+
+#### `executor: JobExecutor` (required)
+`PgBoss` is currently our only job executor, and is recommended for low-volume production use cases. It requires your `app.db.system` to be `PostgreSQL`.
+
+####  `perform: dict` (required)
+
+  - ##### `fn: fn` (required)
+  An `async` JavaScript function of work to be performed. It can optionally take a JSON value as an argument.
+  
+  - ##### `executorOptions: dict` (optional)
+  Executor-specific default options to use when submitting jobs. These are passed directly through and you should consult the documentation for the job executor. These can be overridden during invocation with `submit()` or in a `schedule`.
+
+    - ##### `pgBoss: JSON` (optional)
+    See the docs for [pg-boss](https://github.com/timgit/pg-boss/blob/7.2.1/docs/readme.md#sendname-data-options).
+
+#### `schedule: dict` (optional)
+  
+  - ##### `cron: string` (required)
+  A 5-placeholder format cron expression string. See rationale for minute-level precision [here](https://github.com/timgit/pg-boss/blob/7.2.1/docs/readme.md#scheduling).
+  
+  - ##### `args: JSON` (optional)
+  The arguments to pass to the `perform.fn` function when invoked.
+  
+  - ##### `executorOptions: dict` (optional)
+  Executor-specific options to use when submitting jobs. These are passed directly through and you should consult the documentation for the job executor. The `perform.executorOptions` are the default options, and `schedule.executorOptions` can override/extend those.
+
+    - ##### `pgBoss: JSON` (optional)
+    See the docs for [pg-boss](https://github.com/timgit/pg-boss/blob/7.2.1/docs/readme.md#sendname-data-options).
```

Comment:
Do we have any way to remember updating 7.2.1 here once we update pg-boss? Maybe put a comment in the waspc code where version is specified? Or create an issue for introducing some kind of mechanism to do this for us?

Reply:
I don't have a good way of templating that out. I can at least add a comment near the SemVer for it in Haskell though to update the docs when we change it, good thinking. I can also open a discussion issue on how to handle it.

Reply:
Yeah, I agree figuirng out solution now would be too much work. It might be worth doing it in the future, since there will be multiple libraries we are referring to so tracking all the links could become hard, but for now comment will be enough probably! Great!